fv3core/stencils/c_sw.py

```diff
@@ -246,15 +280,18 @@ def circulation_cgrid(
 
     with computation(PARALLEL), interval(...):
         fx = dxc * uc
+        fx1 = dxc[0, -1] * uc[0, -1, 0]
+
         fy = dyc * vc
+        fy1 = dyc[-1, 0] * vc[-1, 0, 0]
 
-        vort_c = fx[0, -1, 0] - fx - fy[-1, 0, 0] + fy
+        vort_c = fx1 - fx - fy1 + fy
```

Comment:
ah this is necessary, for below, accessing a temporary with an offset. this workaround works? awesome.

Reply:
Yeah, it's that temporaries cannot be referenced with any offsets, but the temporary itself can represent an offset'd calculation :)